### PR TITLE
New package: termbox-1.1.0

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -2803,3 +2803,4 @@ libdouble-conversion.so.1 double-conversion-2.0.1_1
 libspectrum.so.8 libspectrum-1.2.2_1
 libbearssl.so.0 bearssl-0.3_1
 libXfont2.so.2 libXfont2-2.0.1_1
+termbox-1.1.0_1 termbox-1.1.0_1
```
Comment (pullmoll) [inline common/shlibs]: This is definitely wrong. If you have a `-devel` subpackage, there will be some *.so.* file to add here.

**Comment by diogoleal** [comment]
this it okay?

**Comment by Vaelatern** [comment]
No. In the devel package, do not move `.so*` but instead the `.so` files.
And a `common/shlibs` entry is needed, in the form of the others, with the correct versioning information.

**Comment by diogoleal** [comment]
Okay, adjusted!

**Comment by ebfe** [comment]
Where did the the -devel package go?

**Comment by the-maldridge** [comment]
@diogoleal this needs the -devel package back to be mergable.

**Comment by diogoleal** [comment]
@ebfe 

-devel added and branch updated.

**Pull request closed without merging**